src/ls/hover/argument.cr

```diff
@@ -0,0 +1,14 @@
+module Mint
+  module LS
+    class Hover < LSP::RequestMessage
+      def hover(node : Ast::Argument, workspace) : Array(String?)
```

Comment:
why the array may include nulls?

Reply:
Good point, if there's no good reasons to have it, I wanted to suggest using `Array(String)` instead.

Reply:
Because of these: https://github.com/mint-lang/mint/blob/language-server/src/ls/hover/function.cr#L35

Reply:
And why's that needed?

Reply:
Because we want to display the description of the function (or something else) which can be empty, but providing a empty string will add an empty block in the UI, so either we `compact` there or in the main method.

Reply:
I'd say just there. That will cleanup the api and remove the messy handing of nilable `String`.